packages/blade/src/components/Button/BaseButton/buttonTokens.ts

```diff
@@ -1,24 +1,23 @@
 import type { ThemeTokens } from '../../../tokens/theme/theme.d';
 import type { TypographyPlatforms } from '../../../tokens/global/typography';
-import type { Required } from '../../../_helpers/types';
 import type { IconProps } from '../../Icons';
 import type { BaseButtonProps } from './BaseButton';
 
 //TODO: Sort keys in all objects
 
-export type ButtonMinHeight = 48 | 40 | 32 | 28;
+export type ButtonMinHeight = 48 | 36 | 32 | 28;
```

Comment:
```suggestion
export type ButtonMinHeight = 28 | 32 | 36 | 48;
```

nit(sorry OCD triggering): sort these values?